main.py

```diff
@@ -1,15 +1,15 @@
-import aiohttp
 import argparse
 import asyncio
 import os
 
-from src.fetch import GetNewestEvent, SearchEvent, FetchBorder, FetchCover
-from src.make import makefile
+import aiohttp

```

Comment:
Why aiohttp is separate individually,
is because it is the core library?

Reply:
I'm used to formatting imports with the following style:

```text
python builtin libraries

external 3rd-party libraries

programmer defined libraries

...
```

Again, it's just my personal opinion. Maybe you can provide your own styling guide so that whoever wants to contribute to this repo can follow your styling preferences.

Reply:
I see, I'm more prefer your coding style, the organization is more clear now!
appreciated!